test/functionalTest/cases/1080_batch_query_context/batch_query_context_scopes.test

```diff
@@ -146,8 +144,8 @@ echo
 echo
 
 
-echo "08. POST /v2/op/query, with E.* with scopes q:!type and q=A5==45, see E5/none"
```

Comment:
This was simply a mistake in the description of the step. The filter is actually about 'existance of B', it was changed when enity type optionality was revoked ...
Now it is fixed.

NTC